dev/src/lobster/idents.h

```diff
@@ -1380,6 +1380,14 @@ inline string Signature(const NativeFun &nf) {
         FormatArg(r, arg.name, i, arg.type);
     }
     r += ")";
+    if (nf.retvals.size() > 0){
+        r += "->(";
```

Comment:
spaces around `->`

Reply:
I just made it looks similar to definition of user defined functions, which was without spaces around "->" and with brackets around returning tuple. It seems like this code responsive for it
[dev/src/lobster/idents.h](https://github.com/aardappel/lobster/blob/72210a9af1861ae9099336a543d9b979dcb663db/dev/src/lobster/idents.h#L1406C1-L1426C2)

Reply:
Right now in release version of Lobster.
![image](https://github.com/user-attachments/assets/b004b340-08a3-47cd-93e5-9142abb14de7)
![image](https://github.com/user-attachments/assets/7543ca71-731a-4106-ba98-d0a12789dc84)
so should signature format of builtin function differs from user function?

Reply:
Good catch, they should all look like the definition, just changed the error message.